stable/pod-gpu-metrics-exporter/templates/rbac.yaml

```
@@ -0,0 +1,31 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "pod-gpu-metrics-exporter.serviceAccountName" . }}
```

```suggestion
  name: {{ template "pod-gpu-metrics-exporter.fullname" . }}
```
I think this is what you meant